libraries/AP_HAL_SITL/Synth.hpp

```diff
@@ -33,7 +33,7 @@ struct sTone
 ////////////////////////////////////////////////////////////
 // Converts frequency (Hz) to angular velocity
 ////////////////////////////////////////////////////////////
-const double PI = 3.14159265359;
+const double PI = M_PI;
```

Comment:
I think this is someone else's code we're borrowing here, too.  Not sure we want to be modifying it here.

Reply:
I am not sure we should put a float PI into a double PI